test/generate.js

```diff
@@ -8,14 +8,15 @@ const fs = require('fs')
 const https = require('https')
 const semver = require('semver')
 
-const SUPPORTED_FEATURES = new Set([
-  'scoring',
-  'namespaces',
-  'portableText',
-  'experimentalFunctions',
-])
-const GROQ_VERSION = '2.0.0'
-const DISABLED_TESTS = ['Filters / documents, nested 3', 'Parameters / Undefined']
+const SUPPORTED_FEATURES = new Set(['portableText'])
```

Comment:
Curious about why you removed `namespaces`?

Reply:
The tests aren't tagged with `namespaces` anymore since that's implicit by supporting GROQ-1.